pkg/cli/cmds/server.go

```diff
@@ -142,13 +141,37 @@ func ServerRun(clx *cli.Context) error {
 }
 
 func validateCNI(clx *cli.Context) {
-	cni := clx.String("cni")
+	cnis := []string{}
+	for _, cni := range clx.StringSlice("cni") {
+		for _, v := range strings.Split(cni, ",") {
+			cnis = append(cnis, v)
+		}
+	}
+
+	switch len(cnis) {
+	case 0:
+		cnis = append(cnis, "canal")
+		clx.Set("disable", "rke2-multus")
+	case 1:
+		if cnis[0] == "multus" {
+			logrus.Fatal("invalid value provided for --cni flag: multus must be used alongside another primary cni selection")
+		}
```

Comment:
Whoops, also need to disable multus on single arg.
```suggestion
		fallthrough
	case 1:
		if cnis[0] == "multus" {
			logrus.Fatal("invalid value provided for --cni flag: multus must be used alongside another primary cni selection")
		}
		clx.Set("disable", "rke2-multus")
```